app/controllers/filler/basicPokemonDetail.js

```diff
@@ -1,87 +1,31 @@
-const PokemonModel = require(__appbase + 'models/pokemon'),
-    PokemonStore = require(__appbase + 'stores/pokemon'),
-    jsonfile = require('jsonfile'),
-    async = require('async'),
-    cfg = require(__appbase + '../config');
-
 let fs = require('fs'),
```

Comment:
You have used let, const without using `"use strict";` and when I tried running the script it throws me an error.